Specify what is meant by support (#576)
awslabs/aws-lc: README.md

included for x86 and ARM.

## Quickstart for Amazon Linux 2

> AWS-LC’s libcrypto is a C library and needs a C compiler. AWS-LC's libssl is a

Reviewer: Nit:  Should call out libdecript here until we get rid of it?

Author: It's gone

other libcrypto projects.

## AWS-LC features

### API Compatibility

AWS-LC is compatible with the majority of OpenSSL’s APIs to make it easy to use with
existing applications. We’re open to discussing adding missing functionality and
understanding your use case in an Issue or on Gitter.

### Compiler, OS, and CPU support

AWS-LC correctness is tested on a variety of C/C++ compiler, OS, and CPU

> combinations. For a complete list of tested combinations see

Reviewer: Can you please modify the ci/Readme to add Android as a platform?  Have we added other platforms since?

[tests/ci/Readme.md](https://github.com/awslabs/aws-lc/blob/main/tests/ci/README.md).
If you use a different combination and would like to make sure we test it,
please open an issue to discuss adding it to our CI.

### Algorithm optimization support

A portable C implementation of all algorithms is included and optimized assembly
implementations of select algorithms is included for some x86 and Arm CPUs. We
use [AWS Graviton processors](https://aws.amazon.com/ec2/graviton/) to test
ARMv8 optimizations and Intel CPUs to test x86 and x86-64 optimizations.

The [Intel Software Development Emulator](https://software.intel.com/content/www/us/en/develop/articles/intel-software-development-emulator.html)

> is used to run tests all modern x86 processors.

Reviewer 2: ```suggestion
is used to run tests on many different x86 processors.
```
we try to run the tests on all Intel models that came out in the last 15-20 years.

Author: Makes sense
